frontend/src/main/scala/bloop/bsp/RefreshProjects.scala

```diff
@@ -0,0 +1,19 @@
+package bloop.bsp
+
+import bloop.io.AbsolutePath
+import bloop.logging.Logger
+
+import scala.sys.process.ProcessLogger
+import scala.sys.process.Process
+
+object RefreshProjects {
+  def run(workspaceDir: AbsolutePath, command: List[String], logger: Logger): Unit = {
+    val exit = Process(command, cwd = Some(workspaceDir.toFile))
+      .!(ProcessLogger(logger.info, logger.info))
```

Comment:
Can we use nuprocess instead of std's `Process`? This is the standard throughout the codebase to do any kind of system process

Reply:
I used Forker.run api, which uses nuprocess under the hood